### ISSUE TITLE
About the TOKEN

### ISSUE THREAD

**Issue by ntende** [Clevero/dashing-performance-monitor#1]
Could you  shed more light about where to  get your $TOKEN and $HOST variables

**Comment by Clevero** [comment]
Yes, you are right. The setup instructions are not the best. My fault. Will try to make a better one in the next few days! :)
